### PR TITLE
Usagedocs

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

This PR updates the usage documentation.

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR> -stub`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test_one_sample,docker --outdir <OUTDIR> -stub`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/usage.md
Comment (jemten) [inline docs/usage.md:16]: Add smncopynumbercaller

File: docs/usage.md
Comment (jemten) [inline docs/usage.md:36]:
```suggestion
  - [Run sentieon](#run-sentieon)
  - [Azure Resource Requests](#azure-resource-requests)
  - [Running in the background](#running-in-the-background)
  - [Nextflow memory requirements](#nextflow-memory-requirements)
```

File: docs/usage.md
Comment (jemten) [inline docs/usage.md:58]: When this goes to master the `-revision` parameter should reflect that. Can we make it clear somehow

File: docs/usage.md
Comment (jemten) [inline docs/usage.md:67]:
```suggestion
> When you run the command again, it will fetch the pipeline from cache even if a more recent version of the pipeline is available. To make sure that you're running the latest version of the pipeline, update the cached version of the pipeline by including `-latest` in the command.
```

File: docs/usage.md
Comment (jemten) [inline docs/usage.md:38]:
```suggestion
## Introduction
```

Comment (jemten) [reply docs/usage.md:38]: We already have the "main" heading at the top of the document. Same comment on subsequent "main" headings

File: docs/usage.md
Comment (jemten) [inline docs/usage.md:80]:
```suggestion
## Run nf-core/raredisease with your data
```

File: docs/usage.md
Comment (jemten) [inline docs/usage.md:48]:
```suggestion
## Run nf-core/raredisease with test data
```

File: docs/usage.md
Comment (jemten) [inline docs/usage.md:42]:
```suggestion
## Prerequisites
```

File: docs/usage.md
Comment (jemten) [inline docs/usage.md:119]: This we also need to update when this goes to master

File: docs/usage.md
Comment (jemten) [inline docs/usage.md:128]: smncopynumbercaller

File: docs/usage.md
Comment (jemten) [inline docs/usage.md:149]: The line breaks doesn't seem to work on the rendered markdown

File: docs/usage.md
Comment (jemten) [inline docs/usage.md:178]: line breaks not rendered

File: docs/usage.md
Comment (jemten) [inline docs/usage.md:201]: Line breaks. Also some of these paths points to your fork of the test-dataset repo

File: docs/usage.md
Comment (jemten) [inline docs/usage.md:209]: Should we add a sentence or example file for reduced penetrance, score_config and vep_filters?
Maybe also describe that vep_filters will split the results in a "clinical" list and research list.

File: docs/usage.md
Comment (jemten) [inline docs/usage.md:280]:
```suggestion
## Troubleshooting
```

File: docs/usage.md
Comment (jemten) [inline docs/usage.md:264]:
```suggestion
## Best practices
```

File: docs/usage.md
Comment (jemten) [inline docs/usage.md:345]:
```suggestion
## Custom configuration
```

**Review by jemten — COMMENTED** [review]
Very nice Ram! ⭐ 

We should also update the [output document](https://github.com/nf-core/raredisease/blob/dev/docs/output.md) or at least indicate that this section is under development. A touch up to this section on the main readme is also necessary. https://github.com/nf-core/raredisease#pipeline-summary

File: docs/usage.md
Comment (ramprasadn) [reply docs/usage.md:58]: Yupe! Will update the document after we get community approval and before we make an actual release.

**Review by peterpru — COMMENTED** [review]
Looks good to me so far. Will we be mentioning every single program we use in the pipeline? I assume a new user would be interested in all the different output the pipeline generates, and what it means.

**Comment by ramprasadn** [comment]
@peterpru Yes, that information will go into the main README file 😄

File: docs/usage.md
Comment (Lucpen) [inline docs/usage.md:203]:
```suggestion
<sup>4</sup>Gnomad vcf file can be downloaded from [here] (https://gnomad.broadinstitute.org/downloads).<br />
<sup>5</sup>Used by genmod while modeling the variants. Contains a list of loci that show [reduced penetrance](https://medlineplus.gov/genetics/understanding/inheritance/penetranceexpressivity/) in people. Sample file [here](https://github.com/nf-core/test-datasets/blob/raredisease/reference/reduced_penetrance.tsv).<br />
```

**Review by Lucpen — COMMENTED** [review]
Very nice explanation Ram! :D
I only found one tiny thing to change

File: docs/usage.md
Comment (jemten) [reply docs/usage.md:128]: We will add the smncopynumber caller to the main README and output instead

File: docs/usage.md
Comment (jemten) [reply docs/usage.md:213]: done

**Review by jemten — APPROVED** [review]
Nice! 📝 🥳

**Pull request merged**
